src/cartesian/CartesianAxis.tsx

```diff
@@ -37,11 +46,10 @@ export interface CartesianAxisProps {
   hide?: boolean;
   label?: any;
 
-  minTickGap?: number;
+  minTickGap?: TickGap;
```

Comment:
why this type is introduced? so far I see it's only alias for number and  when you will hit suggestion prompt in any IDE imho (personal preference) it's easier to wrap your head around when it just hint you that is a number instead some kind of custom type.
even if there are plans to expand it like for example: 
`TickGap = number | string`  or anything like that - I still would like this notation over a custom type, that I don't know and have to see it's definition first to understand it. 
But maybe there is some specific use case that I don't know about and this solves some other issue?

Reply:
No, good point. I thought this would help storybook pick up comments above the Type from a single place, but it turned out to not work 🤦 

Will remove the alias.